Shared/Extensions/HashExtensions.swift

```diff
@@ -57,18 +57,3 @@ extension Data {
     }
 }
 
-extension Data {
```

Comment:
Unrelated commit, Xcode is warning this function is not memory safe as-written. It is unused.